### PR TITLE
story 1

### PR DESCRIPTION
**Pull request by tking42**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
 <?php
+
+$db = new PDO('mysql:host=db; dbname=shoes', 'root', 'password');
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT `brand`, `name`, `color`, `cost`, `image` FROM `shoes`');
+$query->execute();
+$results = $query->fetchAll();
```
Comment (joseph-padfield) [inline index.php:22]: Could use require once instead?

**Review by joseph-padfield — APPROVED** [review]

File: index.php [hunk 2]
```diff
@@ -1 +1,44 @@
+<html lang="en">
+<head>
+    <title>Crep Collector</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com" rel="stylesheet">
+</head>
+<body>
+<nav>
+    <h1>Trainer Tracker</h1>
```
Comment (matt-ioa) [inline index.php:10]: Are you calling it Crep Collector or Trainer Tracker?

File: index.php [hunk 3]
```diff
@@ -1 +1,44 @@
+<html lang="en">
+<head>
+    <title>Crep Collector</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com" rel="stylesheet">
```
Comment (matt-ioa) [inline index.php:6]: I'm not sure that this is actually going to load a custom font stylesheet - the URL doesn't look long enough to me. Do the fonts work?

File: index.php (hunk 1 above)
Comment (matt-ioa) [reply index.php:22]: Jo's suggestion is a good idea! It would be handy to split this into a separate db.php file.

File: index.php [hunk 4]
```diff
@@ -1 +1,44 @@
...
+$query = $db->prepare('SELECT `brand`, `name`, `color`, `cost`, `image` FROM `shoes`');
+$query->execute();
+$results = $query->fetchAll();
+
+echo "<div class=container>";
+
+foreach ($results as $shoe)
+{
+    echo "<div class=box>";
+    echo '<img src=' . $shoe['image'] . '>' . "\n";
```
Comment (matt-ioa) [inline index.php:29]: The `"\n"` won't do anything in HTML, you need to either use a `<br>` or (preferably) add some margin/padding.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    margin: 0;
+    background-color: cadetblue;
+    color: white;
+    font-family: "Josefin Sans", sans-serif;
+}
+
+nav {
+    background-color: white;
+    color: cadetblue;
+
```
Comment (matt-ioa) [inline style.css:12]: Remove this empty line

File: index.php [hunk 6]
```diff
@@ -1 +1,44 @@
+<html lang="en">
+<head>
+    <title>Crep Collector</title>
+    <link rel="stylesheet" href="style.css">
```
Comment (matt-ioa) [inline index.php:4]: Don't forget to link normalise.css before this

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Good job! Just a couple of changes requested, take a look 👀

**Pull request merged**